controllers/akodeploymentconfig/cluster/cluster_controller_addon_secret.go

```diff
@@ -250,6 +250,18 @@ func (r *ClusterReconciler) patchAkoPackageRefToClusterBootstrap(ctx context.Con
 		}
 	}
 
+	// Add skip deleting ako packageinstall annotation to clusterboostrap if it doesn't exist
+	if _, exist := bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey]; !exist {
+		if bootstrap.Annotations == nil {
+			bootstrap.Annotations = make(map[string]string)
+		}
+		bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += "load-balancer-and-ingress-service, "
+	} else {
+		if !strings.Contains(bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey], "load-balancer-and-ingress-service") {
+			bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += "load-balancer-and-ingress-service, "
```

Comment:
bootstrap.Annotations[akoov1alpha1.TKGSkipDeletePkgiAnnotationKey] += ",load-balancer-and-ingress-service"
The comma should be at the front

Reply:
Thanks! updated